File: 1.Arrays/Java/ReverseArray.java
import java.util.Scanner;

public class ReverseArray {

    public static void main(String[] args) {
        int[] arr = new int[5];
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter five numbers:");

        for (int i = 0; i < arr.length; i++){
            arr[i] = sc.nextInt();
        }

        System.out.println("Reversed array: ");

        for (int i = arr.length-1; i >=0 ; i--){
            System.out.print(arr[i]+" ");
        }
        System.out.println();
        sc.close();
    }
}

File: 1.Arrays/Kotlin/ReverseArray.kt


import java.util.Scanner

fun main(){
    print("Enter Five Numbers: ")
    var sc = Scanner(System.`in`)
    var arr = Array(5) { 0 }

    for(i in 0..arr.size-1){
        arr[i] = sc.nextInt()
    }
    // println("hehe - ${arr.size-1}\n")
    for(i in 4 downTo 0){
        print("${arr[i]} ")
    }
    sc.close()
    println()
}